### DISCUSSION TITLE
Web Search Doesn't Launch

### DISCUSSION THREAD

**Discussion by jhlgh** [Flow-Launcher/Flow.Launcher#795] [Q&A] [answered]
Not sure what may be wrong. Thought I'd start in Discussions instead of Issues as I think it's probably more of an issue with my installation rather than Flow Launcher.

I updated to v1.83 and web searches stopped working. Testing with a google search, I can get search suggestions back to Flow, but when hitting "Enter" or any of the Alt hot keys nothing will launch. No errors in the log file (attached) from what I can tell. Is there anything I should look into?
[2021-11-07.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/7492376/2021-11-07.txt)

Things I've tried:

- Downgrade to 1.82
- Uninstall (removed all AppData files as well) and reinstall 1.83
- Manually edit default browser for web search plugin

**Comment by onesounds** [comment, marked as answer]
1. show me your websearch plugin setting panel.
2. Does the flow close when using the enter or alt?
3. Try specifying a browser executable file to the plug-in. Of course, even if a browser is not specified, it must be opened as the default browser.

**Reply by jhlgh** [reply]
![image](https://user-images.githubusercontent.com/93869501/140648594-4c132c3b-b024-4005-95f0-82f25af65d40.png)
Originally I was using Chrome. I switched to Edge and it's working fine. I tried switching back and forth between Chrome and Edge and Chrome would reliably not work. I swapped to Edge > Vivaldi > Chrome and Chrome is now working again. I'll close this out.

**Reply by taooceros** [reply]
Switching back and forth between edge and chrome makes chrome works?
That's interesting. Are you able to open the url when the browser is empty?

**Reply by jhlgh** [reply]
Now that it's fixed, yes. But before, I could switch between edge and chrome and chrome would reliably fail to launch. Leaving it blank does default to the default browser now per what I set in Windows.
